src/test/kotlin/com/contentful/java/cma/EnvironmentsTests.kt

```diff
@@ -152,28 +152,28 @@ class EnvironmentsTests {
 
```

Comment:
This complete test lends it self to be used to test the redirection from `.branch()` to `.clone()`. Maybe you can copy it and update it to use the `.branch()` method while still assuming the same outcome?